libraries/AC_Avoidance/AC_Avoid.h

```diff
@@ -45,8 +45,13 @@ class AC_Avoid {
      * before the fence/object.
      * Note: Vector3f version is for convenience and only adjusts x and y axis
      */
-    void adjust_velocity(float kP, float accel_cmss, Vector2f &desired_vel_cms, float dt);
+    void adjust_velocity(float kP, float accel_cmss, Vector2f &desired_vel_cms, Vector2f &desired_accel_cmss, Vector2f &rel_save_location_cm, float dt);
```

Comment:
If possible let's explain in the comments what "rel_save_location_cm" is.  I wonder is it "safe" instead of "save"?  sorry to be annoying but perhaps let's call it safe_location_rel_cm.  I guess that it's relative to the vehicle's current target location?  or maybe relative to it's current location..?

Reply:
yep, one of my standard typos, changed to safe_location_rel_cm